setup.py

```diff
@@ -84,7 +84,7 @@ def file_walk_relative(top, remove=''):
     # so 'iris_grib._confirm_iris_mercator_support' performs a runtime version
     # check instead + raises an error if required.
     # TODO: update Iris version and remove runtime checking, in future release.
-    install_requires = ['scitools-iris>=2.0.*', 'eccodes'],
+    install_requires = ['scitools-iris>=2.0.*'],# 'eccodes'],
```

Comment:
As I remember, this is because eccodes isn't compatible with python 3, but I think there was another reason as well.  I will check that and put a comment in.

Reply:
@kaedonkers Well it turns out I don't know.  I do know that some stuff didn't work when eccodes was there before, but now it seems fine.  I've pushed it back up having reinstated eccodes, mostly to see if all the tests still pass.

Reply:
Well, seems fine now.  Maybe I'll see if @dkillick can remember anything about this...